release-notes/WCF-Web-Service-Reference-notes.md

```diff
@@ -1,16 +1,16 @@
-# WCF Web Service Reference Provider
+# WCF Web Service Reference Provider - Release Notes
 
-## Information
-
-* [Usage guide](/Documentation/WCF-Web-Service-Reference-guide.md)
-* [Report an issue](https://github.com/dotnet/wcf/issues/new)
+WCF Web Service Reference connected services provider is now part of Visual Studio 2017. Instructions can be found in the [usage guide](/Documentation/WCF-Web-Service-Reference-guide.md) document. Please [let us know](https://github.com/dotnet/wcf/issues/new) if you run into any issues or have any feedback.
 
 ## Release History
 
-### 1.0.0 - Visual Studio 2017 [v15.5 Preview 1]
+### 10/11/2017 - Visual Studio 2017 v15.5 Preview 1.0 [27009.1.d15rel]
 
-* [Supported features](https://github.com/dotnet/wcf/blob/master/release-notes/SupportedFeatures-v2.0.0.md)
+* WCF Web Service Reference connected services provider is included in VS 2017 starting this release.
```

Comment:
Insert "with" between starting and this.

Reply:
Fixed. Thanks @StephenBonikowsky.